ampl-converter/src/main/java/com/powsybl/ampl/converter/version/ExtendedAmplExporterV1.java

```diff
@@ -0,0 +1,357 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+package com.powsybl.ampl.converter.version;
+
+import com.powsybl.ampl.converter.AmplConstants;
+import com.powsybl.ampl.converter.AmplExportConfig;
+import com.powsybl.ampl.converter.AmplSubset;
+import com.powsybl.ampl.converter.AmplUtil;
+import com.powsybl.ampl.converter.util.NetworkUtil;
+import com.powsybl.commons.io.table.Column;
+import com.powsybl.commons.io.table.TableFormatter;
+import com.powsybl.commons.util.StringToIntMapper;
+import com.powsybl.iidm.network.*;
+import com.powsybl.iidm.network.util.ConnectedComponents;
+import com.powsybl.iidm.network.util.SV;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+import static com.powsybl.ampl.converter.AmplConstants.*;
+
+/**
+ * 1st (V1) extension of BasicAmplExporter, associated with AMPL version 1.1 (exporter id).
+ * The extension adds:
+ *  - A slack bus boolean in the bus table.
+ *  - R, G and B characteristics in tap tables.
+ *  - The regulated bus in generator and static var compensator tables.
+ *
+ * @author Nicolas PIERRE {@literal <nicolas.pierre at artelys.com>}
+ * @author Pierre ARVY {@literal <pierre.arvy at artelys.com>}
+ */
+public class ExtendedAmplExporterV1 extends BasicAmplExporter {
+
+    private static final int SLACK_BUS_COLUMN_INDEX = 8;
+    private static final int TAP_CHANGER_R_COLUMN_INDEX = 4;
+    private static final int TAP_CHANGER_G_COLUMN_INDEX = 6;
+    private static final int TAP_CHANGER_B_COLUMN_INDEX = 7;
+    private static final int GENERATOR_V_REGUL_BUS_COLUMN_INDEX = 14;
+    private static final int STATIC_VAR_COMPENSATOR_V_REGUL_BUS_COLUMN_INDEX = 8;
+
+    public ExtendedAmplExporterV1(AmplExportConfig config,
```

Comment:
I wonder about the name of the class. Should we call it `ExtendedAmplExporterV11` ? 
I get the fact that it is the V1 of the extended exporter but the fact that the name is not completly consistent with the version number may be confusing.

I am not completly sure about my suggestion either. To discuss @p-arvy !

Reply:
I agree with you, I had left the name already used in the PR. Knowing that there aren't yet several extended versions of the ampl export, perhaps ExtendedAmplExporter suffices for now. 

What do you think is best, @So-Fras ?

Reply:
Ok fine by me!